test/data.jl

```diff
@@ -1,5 +1,41 @@
 # Tests of data checking and transformation code
 
+@testset "test data summary" begin
+    buf = IOBuffer()
+
+    @testset "5-bus summary from dict" begin
+        data = PowerModels.parse_file("../test/data/matpower/case5.m")
+        PowerModels.summary(data, io=buf)
+
+        output = String(take!(buf))
```

Comment:
`output = sprint((io,d) -> PowerModels.summary(d, io=io), data)` to replace L7-10 is also possible.

Reply:
An alternative is to implement

```julia
summary(io::IO, data::Dict)
    ....
end

print_summary(data::Any)
    summary(STDOUT, data)
end
```

Then this line will be `output = sprint(PowerModels.summary, data)` instead.

Reply:
If you do change the function name, make sure to update the data doc md to be consistent.